src/starknet-wrappers.ts

```diff
@@ -486,10 +502,6 @@ export class VenvWrapper extends StarknetWrapper {
         super(new StarknetVenvProxy(pythonPath), hre);
     }
 
-    protected override get gatewayUrl(): string {
```

Comment:
This shouldn't be removed. The adaptation only needs to take place if using DockerWrapper. Why would we adapt the URL if the user has Docker Desktop if they aren't even using the plugin in dockerized mode?

Reply:
my bad